test/update-db.test.js

```diff
@@ -17,7 +17,7 @@ afterEach(async () => {
 })
 
 async function chdir (fixture, ...files) {
-  testdir = join(tmpdir(), `browserslist-${ fixture }-${ nanoid() }`)
+  testdir = join(homedir(), `browserslist-test-${ fixture }-${ nanoid() }`)
```

Comment:
I had to do this since `escalade` relies on matching `os.homedir()` to continue searching.

Reply:
We can’t allow test creating files in user’s home directory

Reply:
The only other option I can think of is to create temporary directories within the `test/fixtures` location.

Reply:
What will be extact problems of calling `escalade` inside the `/tmp`?

Reply:
Not to worry – you convinced me. 
Many other projects are going to expect the exact same behavior as `find-up`, so I'm just going to match it. I didn't originally agree with it, but if I'm going to share this module I might as well accommodate others' current needs.

`escalade@2.0` looped so long as `dir.startsWith(homedir)`. So if `os.homedir()` is `/Users/lukeed` then `/tmp/var` fails instantly.

I'm releasing escalade@3.0 right now 😂 I tested it locally and it works with the original test code.